charts/gateway-addons-helm/values.yaml

```diff
@@ -42,6 +42,531 @@ prometheus:
     enabled: false
   kube-state-metrics:
     enabled: false
+    customResourceState:
+      enabled: true
+      # Add (Cluster)Role permissions to list/watch the customResources defined in the config to rbac.extraRules
+      config:
```

Comment:
should this live here or in a template ?

Reply:
Adding more context: 
This config should be added to a file and then path to that file can be passed to kube-state-metrics
Here is an [example](https://github.com/kubernetes/kube-state-metrics/blob/main/docs/metrics/extend/customresourcestate-metrics.md#configuration)

Reply:
We are using the Prometheus chart that comes with the Prometheus add-on. This chart does not seem to have that confg-file option
https://github.com/prometheus-community/helm-charts/blob/995b3392c82ecfdec4cbc432064b04b40c71c8aa/charts/kube-state-metrics/values.yaml#L452

Reply:
looks like the config file option is enabled by default: https://github.com/prometheus-community/helm-charts/blob/995b3392c82ecfdec4cbc432064b04b40c71c8aa/charts/kube-state-metrics/templates/deployment.yaml#L139C47-L139C83 
and when you add those metrics as config it already goes into a [configmap](https://github.com/prometheus-community/helm-charts/blob/995b3392c82ecfdec4cbc432064b04b40c71c8aa/charts/kube-state-metrics/templates/crs-configmap.yaml) so what you have done is sufficient then. :heavy_check_mark: